test/run

```diff
@@ -25,6 +25,11 @@ case $TEST_SCENARIO in
         test/image-prepare --quick --containers --verbose $TEST_OS
         test/containers/run-tests --container ${TEST_SCENARIO#*-}
         ;;
+    dnf-copr*)
+        test/image-prepare --verbose $TEST_OS
+        bots/image-customize -v --run-command 'dnf -y copr enable rpmsoftwaremanagement/dnf-nightly && dnf -y update dnf' $TEST_OS
```

Comment:
Ooh, that's clever! So we don't even need to adjust the bots *.install script for that. :+1: 

I think this should update *all* packages from the COPR, though -- there's also libdnf, plugins, or the daemon in the future. Can we tell dnf update to only consider a particular repo? Or we just update everything -- it may catch a thing or two that isn't in COPR and breaks stuff, but it's still interesting to learn about that as well.

Reply:
> Can we tell dnf update to only consider a particular repo?

`--disablerepo=\* --enablerepo=<copr-stuff>` maybe works?

>  Or we just update everything -- it may catch a thing or two that isn't in COPR and breaks stuff, but it's still interesting to learn about that as well.

Since we run tests right after we create the image there should be very little new packages. So I guess we can go with that?

Reply:
Right, so +1 to just updating the whole thing.

Reply:
I wonder if we should make this more generic -- match on `copr-*)` and put the entire copr name into the scenario. Then we can reuse this trivially to test other COPRs.

Reply:
Fine to do the generalization later, when we have the second copr to test, of course.

Reply:
> I wonder if we should make this more generic -- match on copr-*)

technically yes. But two things:
1. The names would be crazy - `fedora-testing/copr-rpmsoftwaremanagement/dnf-nightly@cockpit-project/cockpit`
2. I am not super sure if currently there can be `/` in the name, I kinda feel like cannot. But that is surely fixable in bots.